### ISSUE TITLE
Logging?

### ISSUE THREAD

**Issue by dreusel** [lesnitsky/flutter_local_assets_server#5]
Hi,
I'm just wondering if there is some option to get some logging in the debug console, mostly about what resources are being requested and which response code was sent? So maybe something like this:
GET /index.html 200
GET /folder-i-forgot-to-include-in-assets/picture.jpg 404

That would be a tremendous help in getting past some of the first hurdles using this :)

**Comment by lesnitsky** [comment]
great idea
adding this functionality shortly

**Comment by lesnitsky** [comment]
not that shortly, but still...
you can now pass an instance of `DebugLogger` to the `LocalAssetsServer`

```dart

final server = new LocalAssetsServer(
  address: InternetAddress.loopbackIPv4,
  assetsBasePath: 'web',
  logger: DebugLogger(), // defaults to SilentLogger
);

**Comment by lesnitsky** [comment]
available in v2.0.1
